horizon/opal_relay_api.py

```diff
@@ -122,19 +138,39 @@ async def send_ping(self):
             ),
         ) as response:
             if response.status != status.HTTP_202_ACCEPTED:
-                text = await response.text()
-                raise RuntimeError(
-                    f"Server responded to token request with a bad status {response.status}: {text}"
+                try:
+                    text = await response.text()
+                except:
+                    text = None
+                raise RelayAPIException(
+                    "relay-api",
+                    response.status,
+                    f"Server responded to token request with a bad status {response.status}: {text}",
```

Comment:
@shaulk i don't think this error message is correct.
this looks like a copy-paste issue.

also it will create a super verbose message instead of simply showing the status code we got.

Reply:
Fixed.